scripts/load_i18n.py

```diff
@@ -3,7 +3,7 @@
 import sys
 
 KEYS = [
-    "city-bureau",
+    "site-owner",
```

Comment:
again, just generally assuming this parameter is meant to be 'people who are responsible for this site and its contents :-)